ci/docker/buildenv/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y git build-essential  autoconf flex bison \
+                                         gcc-6 gcc-5 gcc-4.8 g++-6 g++-5 g++-4.8 \
+                                         bc perl python3 gdb
+
+RUN apt-get install -y wget
+
+WORKDIR /tmp
+
+ADD build-systemc.sh /tmp/
+RUN ./build-systemc.sh
+
+RUN cpan install -fi Unix::Processors Parallel::Forker Bit::Vector
+
+RUN git clone https://github.com/veripool/vcddiff.git && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use WORKDIR to switch to a directory](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)